### ISSUE TITLE
[Crash] AirSim Unity - Unity 2018.2.15f1 

### ISSUE THREAD

**Issue by TreyYi** [Microsoft/AirSim#1546]
Hi,

I've tried to run the sample scenes that MS provides, but it crashes Unity.
I loaded the scenes such as Car, Drone, but as soon as I click the Play button, it crashes Unity.
![image](https://user-images.githubusercontent.com/32972219/48537666-9c32b300-e8f5-11e8-940d-c6e7be574e27.png)
![image](https://user-images.githubusercontent.com/32972219/48537716-b8365480-e8f5-11e8-9b3f-e1bf6fae4ace.png)

Any suggestion?

Regards,
Trey

**Comment by dza6549** [comment]
Hey Trey

my Unity crashed when using Win 10 SDK 10.0.17763.1.
10.0.15063.0 loaded but did not work :(  You saw my post.

You can check your target:
1: Open ..\AirSim\Unity\AirLibWrapper\AirsimWrapper.sln with VS
2: In VS open Project .. Properties
3: What do you have?

![treysdkversion](https://user-images.githubusercontent.com/27390320/48538597-ff7a1080-e908-11e8-9255-285adde465dc.png)

**Comment by TreyYi** [comment]
> AirsimWrapper

I have these options for Windows SDK version
![untitled](https://user-images.githubusercontent.com/32972219/48539155-b1a9dc00-e8f9-11e8-9691-e87200ceb49a.png)

However, when I typed the command below on PowerShell to check what version I have,
$(Get-Item "hklm:\SOFTWARE\Microsoft\Microsoft SDKs\Windows").GetValue("CurrentVersion")
![image](https://user-images.githubusercontent.com/32972219/48539214-dc943000-e8f9-11e8-8905-b5e918bf7760.png)
My current version is 8.0.50727.

Please let me know if I can provide you with any additional information.

**Comment by TreyYi** [comment]
Thanks for your prompt reply.
Though, It's still not working with different Windows SDK settings.
It just crashes Unity Application as soon as I click "Play" button..

**Comment by dza6549** [comment]
Yeah. It is only 24 hours old. :)

Maybe we should wait until someone gets it working.

Cheers!

**Comment by madratman** [comment]
@TreyYi  @dza6549  please use win sdk **8.1**, [Visual Studio 2017, 15.8.9](https://docs.microsoft.com/en-us/visualstudio/productinfo/installing-an-earlier-release-of-vs2017) and try again. 
sdk 8.1 was mentioned in the docs before. VS 2017 15.9.0 was released on Nov 13, and that is new. I have filed a PR an update to the doc. till than, you can look at the diff https://github.com/Microsoft/AirSim/pull/1549/files

**Comment by dza6549** [comment]
@madratman Ah I see - I will update VS Community - I am using 15.65

Thanks

**Comment by madratman** [comment]
@dazhang88 @TreyYi My bad. I think anything <=15.8.9 would work. I think your issue is not using windows sdk 8.1. Can you try with 15.65 + 8.1 and get back to us?

**Comment by dza6549** [comment]
Just a heads up to anyone who is following this thread. The download links on the VS 2017 15.8.9 page mentioned above appear to be mixed up. Community.exe downloads vs_enterprise.exe and vice versa.

**Comment by dza6549** [comment]
@madratman whoops missed your comment. And have uninstalled 15.6.5 already :)

So 15.8.9 it is :) Cheers

**Comment by dza6549** [comment]
@TreyYi 

I got it sorta working over here [https://github.com/Microsoft/AirSim/issues/1550](https://github.com/Microsoft/AirSim/issues/1550)

See if this helps

Cheers!

**Comment by dza6549** [comment]
@TreyYi Closing the thread is ok with me man. It is your thread! :)

**Comment by TreyYi** [comment]
Yup you right. Thanks! Hope the Drone issue would be resolved soon. I'll close the thread.

**Comment by sab2018** [comment]
Windows 7 OS,
Unity 2018.2.14
VS 15.6.7

Just had a question on the SDK. I've tried all the solutions mentioned here. Was running VS 15.9. The build didn't succeed because I had PowerShell 3.1. Upgraded to 5.1. Then tried to rebuild with VS 15.6,7 + 8.1 SDK. had to get SDK 16299 as well after I got an error for it while building the unity project. Build succeeded but I'm getting the same screen as shown by @TreyYi  on unity and unity crashes as I press play. 
@dza6549 @madratman My target platform in VS is windows 10, with SDK version 16299. is that fine for a windows 7 OS?  maybe that's the reason my project crashes? I can't seem to find the cause for this.
